### PR TITLE
[Pre-WIP, Do Not Review] Treasure Peninsula

### PR DESCRIPTION
**Pull request by jMyles**
* #2190 
* #2189
* #2188 
* #2187
* #2203
* Unskip tests
* #2201
* #2227 
* #2228

### CODE REVIEW

File: nucypher/characters/unlawful.py [hunk 1]
```diff
@@ -61,15 +65,18 @@ def from_target_ursula(cls,
...
 
         if attach_transacting_key:
-            cls.attach_transacting_key(blockchain=target_ursula.blockchain)
+            cls.attach_transacting_key(blockchain=target_ursula.policy_agent.blockchain)
 
 
         vladimir = cls(is_me=True,
                        crypto_power=crypto_power,
                        db_filepath=cls.db_filepath,
+                       domains=[':TEMPORARY_DOMAIN:'],
```
Comment (derekpierre) [inline nucypher/characters/unlawful.py:77]: We should use the ``TEMPORARY_DOMAIN` constant in `nucypher/config/constants.py`. #2207 makes the value of the constant for this domain lowercase

File: nucypher/datastore/datastore.py [hunk 2]
```diff
@@ -84,7 +84,7 @@ class Datastore:
 
     # LMDB has a `map_size` arg that caps the total size of the database.
     # We can set this arbitrarily high (1TB) to prevent any run-time crashes.
-    LMDB_MAP_SIZE = 1_000_000_000_000
+    LMDB_MAP_SIZE = 10_000_000
```
Comment (derekpierre) [inline nucypher/datastore/datastore.py:87]: We only wanted the lower map size for tests no?

File: nucypher/characters/unlawful.py (hunk 1 above)
Comment (cygnusv) [reply nucypher/characters/unlawful.py:77]: Good eye!

Comment (jMyles) [reply nucypher/characters/unlawful.py:77]: Please bring these comments over to #1741.

**Comment by derekpierre** [comment]
@KPrasch did we intend to merge this PR, I thought it was WIP?

**Comment by KPrasch** [comment]
Hmm - No, not intentional.  Unsure how that even happened... Perhaps the histories were identical?

**Pull request merged**